setup-env.sh

```diff
@@ -311,6 +311,7 @@ if [ -z "$RUN_IN_CONTAINER" ]; then
     -e RLWRAP_HOME=$(pwd) \
     -e RUN_IN_CONTAINER=true \
     -e MIGRATOR_ALLOW_STOP_PATH="${MIGRATOR_ALLOW_STOP_PATH:-false}" \
+    -e HOST_IP=$(hostname -I | cut -d' ' -f1) \
```

Comment:
You should merge integration branch in to oneui before. Because this line was removed

Reply:
So should I create a new PR with that merge and deal with that first?

Reply:
yes

Reply:
or you can just bump new image version in to integration branch. It would be much easier